src/execution/execute.ts

```diff
@@ -2052,8 +2056,8 @@ async function executeStreamIterator(
 
 function filterSubsequentPayloads(
   exeContext: ExecutionContext,
-  nullPath?: Path,
-  currentAsyncRecord?: AsyncPayloadRecord,
+  nullPath: Path | undefined,
+  currentAsyncRecord: AsyncPayloadRecord | undefined,
```

Comment:
It looks like every call has both args, can you just make them non-optional?

Reply:
done

Reply:
The nullPath is always passed, and so we can remove the possibility of undefined there, but the currentAsyncRecord could be undefined if it is the initial response that is erroring, and so we can make it non-optional, but we cannot remove the possibility of undefined.